--- sgml-common-0.5/bin/install-catalog.quiet	Wed Jan 24 09:22:49 2001
+++ sgml-common-0.5/bin/install-catalog	Wed Jan 24 09:24:54 2001
@@ -130,7 +130,7 @@
 	echo "`basename $0`: addition of $SGML_ORDINARY in $SGML_CENTRALIZED"
 	if grep -q $SGML_ORDINARY $SGML_CENTRALIZED 2>/dev/null
 	then
-	  echo "Warning: $SGML_ORDINARY is already installed in the centralized catalog $SGML_CENTRALIZED" >&2
+	  echo "Warning: $SGML_ORDINARY is already installed in the centralized catalog $SGML_CENTRALIZED"
 	else
 	  echo "$SGML_POINTER $SGML_ORDINARY" >> $SGML_CENTRALIZED
 	fi
@@ -148,7 +148,7 @@
           sed -e "\:$SGML_POINTER $SGML_ORDINARY:d" < $SGML_CENTRALIZED > ${SGML_CENTRALIZED}.new
           mv ${SGML_CENTRALIZED}.new $SGML_CENTRALIZED
 	else
-	  echo "Warning: $SGML_ORDINARY was not found in the centralized catalog $SGML_CENTRALIZED" >&2
+	  echo "Warning: $SGML_ORDINARY was not found in the centralized catalog $SGML_CENTRALIZED"
 	fi
 	if [ ! -s $SGML_CENTRALIZED ]
 	then
