--- sgml-common-0.5/bin/install-catalog.oldsyntax	Fri Dec  8 15:10:07 2000
+++ sgml-common-0.5/bin/install-catalog	Wed Jan 10 16:01:18 2001
@@ -42,12 +42,20 @@
    -r|--remove) SGML_ACTION="removal"
 		SGML_CENTRALIZED=$2
 		SGML_ORDINARY=$3
+		if [ -z "$3" -o "$3" = "--version" ]
+		then
+			echo "install-catalog: Old syntax; doing nothing"
+			exit 0
+		fi
 		;;
    -h|--help)	echo -e $SGML_HELP_MESSAGE
 		exit 0
 		;;
    -v|--version) echo -e $SGML_VERSION_MESSAGE
 	 	exit 0
+		;;
+   --install)   echo "install-catalog: Old syntax; doing nothing"
+		exit 0
 		;;
    *)		echo -e $SGML_HELP_MESSAGE >&2
 		exit 1
